buildbot-host/buildmaster/master.cfg

```diff
@@ -157,9 +157,9 @@ c['change_source'].append(changes.GitPoller(repourl = ovpn_dco_repo_url,
 
 c['builders'] = []
 
-# Allow only two concurrent builds across all latent docker workers
+# Limit concurrent builds across all latent docker workers
 cpus = multiprocessing.cpu_count()
-docker_build_lock = util.MasterLock("docker", maxCount=cpus)
+docker_build_lock = util.MasterLock("docker", maxCount=2*cpus)
```

Comment:
Looks good if buildbot can handle this. The original problem was not only CPUs getting choked, but also memory running out with too many concurrent builds.